samples/nextjs/package.json

```diff
@@ -75,16 +81,17 @@
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-yaml": "^0.2.0",
-    "graphql.macro": "~1.4.2",
```

Comment:
Should also be able to remove "graphql-tag" under dependencies

Reply:
Removed